tests/test_codex_importer.py

```
@@ -0,0 +1,102 @@
+    assert result.attachments_dir is not None
+    attachment_files = list(result.attachments_dir.iterdir())
+    assert attachment_files, "expected an attachment file"
+    payload = attachment_files[0].read_text(encoding="utf-8")
+    assert output_lines in payload
+    stats = result.document.stats
+    assert stats["chunkCount"] >= 2
+    assert stats["modelTurns"] >= 0
+    assert stats["userTurns"] >= 1
+
+    # ensure attachment metadata references the saved file
+    links = [info for info in result.document.attachments]
```

Ruff (0.15.9)

[warning] Unnecessary list comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)